lib/api/uikit/class.php

```diff
@@ -148,37 +170,48 @@ function get_less_directories( $type ) {
 		}
 
 		return $directories;
-
 	}
 
 	/**
-	 * Get JS directories.
+	 * Get JavaScript directories.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $type Type.
+	 *
+	 * @return array
 	 */
-	function get_js_directories( $type ) {
+	public function get_js_directories( $type ) {
 
-		if ( 'add-ons' == $type ) {
+		if ( 'add-ons' === $type ) {
 			$type = 'components';
 		}
 
-		// Define uikit src directory.
+		// Define the UIkit src directory.
 		return array( BEANS_API_PATH . 'uikit/src/js/' . $type );
-
 	}
 
 	/**
 	 * Get components from directories.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param array  $components Array of UIkit Components.
+	 * @param array  $directories Array of directories containing the UIkit Components.
+	 * @param string $format File format.
+	 *
+	 * @return array
 	 */
-	function get_components_from_directory( $components, $directories, $format ) {
+	public function get_components_from_directory( $components, $directories, $format ) {
 
-		$extension = ( 'styles' == $format ) ? 'less' : 'min.js';
+		$extension = ( 'styles' === $format ) ? 'less' : 'min.js';
```

Comment:
The parentheses are not needed here.  Let’s standardize on removing them in a ternary.